src/FSLibrary/MissionSlowNodesNetwork.fs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module MissionSlowNodesNetwork
+
+open StellarCoreSet
+open StellarMissionContext
+open StellarFormation
+open StellarStatefulSets
+open StellarSupercluster
+open StellarCoreHTTP
+open StellarNetworkData
+
+let slowNodesNetwork (context: MissionContext) =
+    // Add pessimistic 100ms latency delay to all nodes to further stress the network
+    let context =
+        { context.WithNominalLoad with
+              installNetworkDelay = Some(true)
+              flatNetworkDelay = Some(100)
+              numAccounts = 5000
+              numTxs = 40000
+              coreResources = MediumTestResources }
+
+    let coreSet =
+        MakeLiveCoreSet
+            "fast-core"
+            { CoreSetOptions.GetDefault context.image with
+                  nodeCount = 7
+                  nodeLocs = Some([ Ashburn; Brussels; Beauharnois; Bengaluru; Chennai; Clifton; Columbus ])
+                  accelerateTime = false
+                  dumpDatabase = false }
+
+    let slowCoreSet =
+        MakeLiveCoreSet
+            "slow-core"
+            { CoreSetOptions.GetDefault context.image with
+                  nodeCount = 3
+                  nodeLocs = Some([ CouncilBluffs; Falkenstein; Frankfurt ])
+                  accelerateTime = false
+                  // 2 ms of delay per grain-of-work in core
+                  addArtificialDelay = Some(2000)
```

Comment:
Just to make sure I understand: this sleeps for 2000ms per ledger, and since each ledger has ~1000 transactions (200 tx/s), that's 2ms per transaction. Is that right?

Reply:
Ah, my comment wasn't super clear, the delay we configure in core is in microseconds, so here I'm adding 2000 microseconds=2ms to each work task that the scheduler executes. I'll make the comment clearer!